lab-dustin/data/notes/69cfbbe0-89e4-11e7-91fd-591ba3748fe5.json

```diff
@@ -0,0 +1 @@
+{"id":"69cfbbe0-89e4-11e7-91fd-591ba3748fe5","note":"this is a note"}
```

Comment:
It's a good idea to ignore data directories to avoid this kind of churn. I did so in our class [demo](https://github.com/DeltaVCode/cr-401js-2017/blob/master/09-vanilla_api_persistence/.gitignore), but didn't specifically call it out.